tests/components_to_test/repeated_computed_layer.py

```diff
@@ -36,9 +36,15 @@ def generate(self):
 
 @non_distributed_component_funcs.register(name='repeated_computed_layers')
 def get_training_components():
-    model = NetWithRepeatedlyComputedLayers(checkpoint=True)
+
+    def model_builder(checkpoint=True):
+        return NetWithRepeatedlyComputedLayers()
```

Comment:
`checkpoint` is not passed to `NetWithRepeatedlyComputedLayers`

Reply:
DONE